### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by YogurtTheHorse**
Hey! Really nice console. I found several problems running it, but after all I fixed them and decided to open a pull request, so no-one would face them again :)

### CODE REVIEW

**Comment by popcron** [comment]
Thank you, could you also increment the package version in the json file at the root?
Also about the string concatenation usage, its more efficient to let the StringBuilder append the strings on its own than having strings be created due to a `+` operation. Ill merge after the package version increments, cheers.

**Comment by YogurtTheHorse** [comment]
Yeah, string concatenation could be pretty slow. I'm not sure does it really affect such functionality, but nvm I optimized it as you asked. Package version is also increased.

**Comment by popcron** [comment]
Yeah just want to avoid alloc when possible, looks good now for the most part, just the few lines where i left a comment should be checked over then it should be good to merge.

**Comment by YogurtTheHorse** [comment]
What comments do you mean?

**Comment by popcron** [comment]
Made the comments but forgot to submit them, oops, here they are

File: Runtime/CommandsBuiltin.cs [hunk 1]
```diff
@@ -67,29 +68,33 @@ public static string Owners()
...
 
             foreach (Owner.OwnerMember property in owner.properties)
             {
                 if (property.canSetProperty)
                 {
-                    text.Append(Indent + Indent + property.name);
-                    text.AppendLine(" [value]");
+                    text.Append(TwiceIndent);
+					text.Append(property.name);
+					text.Append(" [value]");
```
[the remark this answers is not in the document]
Comment (YogurtTheHorse) [reply Runtime/CommandsBuiltin.cs]: done

File: Runtime/CommandsBuiltin.cs [hunk 2]
```diff
@@ -9,6 +9,7 @@
 public class CommandsBuiltin
 {
     private const string Indent = "    ";
+    private const string TwiceIndent = TwiceIndent;
```
[the remark this answers is not in the document]
Comment (YogurtTheHorse) [reply Runtime/CommandsBuiltin.cs]: done

[the remark this answers is not in the document]
Comment (YogurtTheHorse) [reply Runtime/CommandsBuiltin.cs]: forgot to test and left it like that, sorry

[the remark this answers is not in the document]
Comment (YogurtTheHorse) [reply Runtime/CommandsBuiltin.cs]: also I believe two appends is over engineering

[the remark this answers is not in the document]
Comment (popcron) [reply Runtime/CommandsBuiltin.cs]: Well either works, so it's all good suppose

**Pull request merged**